OIPA/iati/models.py

```diff
@@ -53,6 +53,7 @@ class Activity(models.Model):
     hierarchy = models.SmallIntegerField(choices=hierarchy_choices, default=1, null=True)
     # last_updated_datetime = models.CharField(max_length=100, default="")
     # value_date = models.DateField(null=True)
+    last_updated = models.DateTimeField(null=True, auto_now=True)
```

Comment:
it is the last time the object was updated ,
the other one is for the last time the iati source was updated.
u tas said it's not necessary anymore and I'll rmove it.


On Wed, Oct 14, 2015 at 11:59 AM, Vincent <notifications@github.com> wrote:

> In OIPA/iati/models.py
> <https://github.com/openaid-IATI/OIPA/pull/235#discussion_r41975057>:
>
> > @@ -53,6 +53,7 @@ class Activity(models.Model):
> >      hierarchy = models.SmallIntegerField(choices=hierarchy_choices, default=1, null=True)
> >      # last_updated_datetime = models.CharField(max_length=100, default="")
> >      # value_date = models.DateField(null=True)
> > +    last_updated = models.DateTimeField(null=True, auto_now=True)
>
> I don't mind it being added to the model if necessary, but with the
> current name it will be confusing with both the last_updated and
> last_updated_datetime in there. Even for me I still don't know what this
> date represents atm; Is it the last time the search index was updated to
> include this activity? Or the last time the parser updated this activity?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openaid-IATI/OIPA/pull/235/files#r41975057>.
>